config/locales/en.yml

```diff
@@ -35,7 +35,7 @@ en:
         created: "Created %{name}"
         error_creating: "Sorry, there was an error creating the feature."
         updated: "Updated %{name}"
-        error_updating: "Could not update feature #%{id}: %{errors}."
+        error_updating: "Could not update %{name}: %{errors}."
```

Comment:
this was done because @feature.to_s => "Feature 57" and this came out as:

Could not update feature Feature 40: ....

now it just says

Could not update Feature 40: ....